chore: test solution (#6)
matter-labs/tutorials: tests/README.md

![Gradient Banner](https://github.com/matter-labs/tutorials/assets/10233439/8efffb9b-ad1f-4bf2-8f73-9cab8f7ccd22)

# zkSync Tutorials

> This document contains an instruction of the test infrastructure preparation/running tests locally.

Reviewer: ```suggestion
This document provides instructions for preparing the test infrastructure and running tests locally.
```

Reviewer 2: ![googleads_mobile_ios_swift_adaptive_banner_example.zip](https://github.com/matter-labs/tutorials/files/13927812/googleads_mobile_ios_swift_adaptive_banner_example.zip)

## The environment preparation

> To have a possibility of test execution there needs to install a local node (L2) based on the next repository https://github.com/matter-labs/era-test-node.

Reviewer: ```suggestion
To execute tests, you need to install a local node (L2) from the following repository: https://github.com/matter-labs/era-test-node
```

Reviewer 2: Yas[]()[]()https://zerion.io/premium

Reviewer 2: @![Screenshot 2024-01-05 at 11.29.17.png](https://github.com/matter-labs/tutorials/assets/150193026/9ef6889b-beb4-4d62-badc-3507e63f70fc)
